provisioning/roles/php-memcached/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
+  - { role: php-fpm }
```

Comment:
Why do you need this dependency? What if someone wants to use php-apache instead?